config.yaml

```diff
@@ -42,6 +42,14 @@ options:
     description: |
       The organization runner group to register the self-hosted runner under. This has no effect on
       runners under a repository.
+  labels:
+    type: string
+    default: ""
+    description: |
+      Additional comma separated labels to attach to self-hosted runners. By default, the labels
+      "self-hosted", application name (default: "github-runner"), 
```

Comment:
Is the line break at the correct place? It looks a bit weird.

Reply:
Right, the line break here is intended to keep `architecture (i.e. "x64", "arm64")` in the same line - otherwise, it would be 
```
...architecture
(i.e. "x64", "arm64"), os (i.e. "linux"), os-flavor (i.e. "jammy") are set. ...
```
which I thought would be a little harder on the eyes.